src/Pages/ChosenProtocol.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import { LineChart, Line, CartesianGrid, XAxis, YAxis } from 'recharts';
+import { useNavigate } from "react-router-dom";
+import { chosenContracts, chosenProtocolName } from "./LeaderBoard"; //import { chosenContracts, chosenVaRofContracts } from "./LeaderBoard"
```

Comment:
- 'chosenContracts' are the contracts for the protocol that is clicked on by the user on the leader board page
- 'chosenProtocolName' is the name of the protocol that the user has clicked on